pkg/manifest/manifest.go

```diff
@@ -72,8 +67,11 @@ type Manifest interface {
 	WriteConfig(out io.Writer) error
 	PlatformConfigFileName() (string, error)
 	WritePlatformConfig(out io.Writer) error
+	WriteResourceLimitsConfig(out io.Writer) error
 	GetLaunchableStanzas() map[launch.LaunchableID]launch.LaunchableStanza
 	GetResourceLimitsStanza() ResourceLimitsStanza
```

Comment:
You can get rid of `GetResourceLimitsStanzas()` (and the accompanying implementation) since you've added `GetResourceLimits()`